From c86e82d833ed60640289274eee0ec24e4820f750 Mon Sep 17 00:00:00 2001
From: Peng Zhou <benjo.zhou@rock-chips.com>
Date: Fri, 20 Oct 2017 17:04:28 +0800
Subject: [PATCH] ARM64: rockchip_defconfig: Enable cif controller

Enable:	ROCK_CHIP_SOC_CAMERA
	ROCKCHIP_CAMERA_SENSOR_INTERFACE
	RK30_CAMERA_ONEFRAME

Change-Id: I965f96fb4a5a40b03b74df05ec5eb00caa1f19c6
Signed-off-by: Peng Zhou <benjo.zhou@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 41e0797315db..92418ced9675 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -457,6 +457,9 @@ CONFIG_CAMSYS_DRV=y
 CONFIG_FLASHLIGHT=y
 CONFIG_LEDS_RT8547=y
 CONFIG_CAMSYS_MRV=y
+CONFIG_ROCK_CHIP_SOC_CAMERA=y
+CONFIG_ROCKCHIP_CAMERA_SENSOR_INTERFACE=y
+CONFIG_RK30_CAMERA_ONEFRAME=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
-- 
2.35.3

